api/managementpb/dbaas/common.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package dbaas.v1beta1;
+
+option go_package = "dbaasv1beta1";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// RunningOperation respresents a long-running operation.
+message RunningOperation {
+  // Progress from 0.0 to 1.0; can decrease compared to the previous value.
+  float progress = 1;
+  // Text describing the current operation progress step.
+  string message = 2;
```

Comment:
Just wondering whether timestamp will be useful here. As an example: for logging, delaying progress, displaying time diff, etc.

Reply:
There is no intention to use that API for logs – it is a current status indicator